packages/paste-website/src/components/shortcodes/table-of-contents/TableOfContentsListItem.tsx

```diff
@@ -1,21 +1,21 @@
 import {styled, themeGet} from '@twilio-paste/styling-library';
 import type {TableOfContentsListItemProps} from './types';
 
-export const marginTopPartial = ({depth}: TableOfContentsListItemProps): string => {
+export const marginTopPartial = ({depth}: TableOfContentsListItemProps): any => {
```

Comment:
By adding the types package themeGet is retyped as returning `any` which is kind of annoying, but I get it.